google/auth/app_engine.py

```diff
@@ -110,3 +110,8 @@ def with_scopes(self, scopes):
     @_helpers.copy_docstring(credentials.Signing)
     def sign_bytes(self, message):
         return app_identity.sign_blob(message)
+
+    @property
+    @_helpers.copy_docstring(credentials.Signing)
+    def signer_email(self):
+        return self.service_account_email
```

Comment:
Pylint won't import the code and check `my_obj.__doc__`, does static analysis pick up the fact that `__doc__` gets set by your decorator?